Examples/src/components/Examples/FeaturedApps/ScientificCharts/LiDAR3DPointCloudDemo/angular.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, AfterViewInit, ViewChild, ElementRef,ChangeDetectorRef } from '@angular/core';
+import { SciChart3DSurface } from 'scichart';
+import { drawExample, drawHeatmapLegend } from './drawExample'; 
+
+@Component({
+  selector: 'app-li-dar3d-point-cloud',
+  template: `<div class="chart-container">
+  <div class="chart-wrapper">
+    <scichart-angular
+      #sciChartSurface
+      [initChart]="initSciChartSurface"
+      style="flex: 1; flex-basis: 50%;">
+    </scichart-angular>
+    <scichart-angular
+      #heatmapLegend
+      [initChart]="initHeatmapLegend"
+      style="position: absolute; height: 90%; width: 100px; top: 0; right: 75px; margin: 20px;">
+    </scichart-angular>
+  </div>
+</div>
+`,
+ 
+})
+export class LiDAR3DPointCloudDemoComponent implements AfterViewInit {
+    @ViewChild('sciChartSurface', { static: false }) sciChartSurfaceRef!: ElementRef;
+    @ViewChild('heatmapLegend', { static: false }) heatmapLegendRef!: ElementRef;
+
+  private sciChartSurface: SciChart3DSurface | undefined;
+  constructor(private cd: ChangeDetectorRef) {}
+
+  async ngAfterViewInit() {
+
+    this.cd.detectChanges();
+
+    setTimeout(async () => {
+        await this.initializeCharts();
```

Comment:
This looks unnecessary. `scichart-angular` already does the initialization internally